ui_framework/src/global_styling/reset/_reset.scss

```diff
@@ -52,13 +53,13 @@ body {
 
 a {
   text-decoration: none;
+
   &:hover {
-    text-decoration: underline;
+    text-decoration: none;
   }
 
   &:focus {
-    text-decoration: underline;
-    background: $kuiFocusBackgroundColor;
+    text-decoration: none;
```

Comment:
That's a valid point, but man that mixin is going to be everywhere.

I'm not too opinionated, but if you go the mixin route make sure to add it back to the breadcrumbs.